content/F2017/F17-event-options-panel.md

```diff
@@ -0,0 +1,11 @@
+Title: Fall 2017 Options Panel
+Date: 2017-11-08
+Category: Events
+Tags: social, panels
+Slug: options-panel-F17
+Author: Ruofan Luo
+Summary: Ask questions about options in CS
+
+Explore exciting options in CS! No idea what courses you want to take? Not sure what to do with your CS passions? Come to our Advisor Panel to find out! Talk to upper years and advisors about the options available in the CS major, as well as joint degrees and related programs.
+
+This is a women-only event.
```

Comment:
For consistency ... need the 
Who
What
When 
see http://wics.uwaterloo.ca/2017/03/dinner-with-professors-W17/ for example

dont forget to link the faq for women only